app/actors/hyrax/actors/file_actor.rb

```diff
@@ -87,7 +87,9 @@ def perform_ingest_file_through_valkyrie(io)
         create_version(file_metadata, user)
         id = file_metadata.file_identifier
         file_set.file_ids << id
+        file_set.original_file_id = id
         Hyrax.persister.save(resource: file_set)
+        Hyrax.publisher.publish('object.metadata.updated', object: file_set, user: user)
```

Comment:
Not convinced we need to do the indexing here.  It will be done again after the ingest is complete.  This one creates a super small solr doc because most of the fields are populated from the file metadata.

Reply:
if `file_set`'s metadata is updating, i think publishing the event is the right idea. we want listeners to be able to handle events as they prefer, and shouldn't try to regulate behavior by declining to publish.

is your concern about performance due to unneeded index calls/commits? or something else?